manifests/pipecd/templates/deployment.yaml

```diff
@@ -98,6 +98,9 @@ spec:
           - --enable-grpc-reflection={{ .Values.server.args.enableGRPCReflection }}
           - --encryption-key-file={{ .Values.secret.mountPath }}/{{ .Values.secret.encryptionKey.fileName }}
           - --log-encoding={{ .Values.server.args.logEncoding }}
+{{- if .Values.server.args.metrics }}
+          - --metrics=true
+{{- end }}
```

Comment:
I think `- --metrics={{ .Values.server.args.metrics }}` is just fine, since we have parse the value here already 👀  https://github.com/pipe-cd/pipe/blob/master/pkg/cli/app.go#L108-L113

Reply:
for sure. Piped does as well.

Reply:
👍